cypress/e2e/registration.cy.js

```diff
@@ -1,11 +1,75 @@
 /// <reference types='cypress' />
-
+import { generateUser } from '../support/generate';
 describe('Student Registration page', () => {
   before(() => {
-
+    cy.visit('https://demoqa.com/automation-practice-form');
   });
 
-  it('', () => {
+  const { email, firstName, lastName, address } = generateUser();
+  const number = 1234567891;
+
+  it('should enter vaild data to register', () => {
+    cy.get('#firstName').type(firstName);
+
+    cy.get('#lastName').type(lastName);
+
+    cy.get('#userEmail').type(email);
+
+    cy
+      .get('#genterWrapper > .col-md-9 > :nth-child(1) > .custom-control-label')
+      .click();
```

Comment:
Using direct child selectors (>) makes your tests brittle because they will break if the structure of your HTML changes. Try to use data-* attributes to select elements, as they are less likely to change than CSS or XPath selectors.